core/src/graphics/Renderer.ts

```diff
@@ -21,7 +21,7 @@ export class Renderer {
     // create the pixi.js application
     this.app = new Application({
       view: props.canvas,
-      resolution: 1,
+      resolution: 2, // TODO configurable
```

Comment:
noticed game looked much nicer rendering at 2x resolution. need to keep default 1 for performance but it should be configurable